scss/_variables.scss

```diff
@@ -244,6 +244,8 @@ $link-hover-decoration: underline !default;
 // Darken percentage for links with `.text-*` class (e.g. `.text-success`)
 $emphasized-link-hover-darken-percentage: 15% !default;
 
```

Comment:
should these vars go here or in the new _links.scss file

Reply:
Hmm, that's a good question, and one I'm not sure of the answer to. Not sure what is considered "best practice" for that. I assume all vars should live in the _variables file. 🤷‍♂️

Reply:
will keep it there for now, we have other vars we've added to this file. but when we upgrade we will have to resolve conflicts in this file. i think best practice is to have an override.scss